server/src/main/java/io/druid/segment/indexing/RealtimeTuningConfig.java

```diff
@@ -40,6 +40,7 @@
 public class RealtimeTuningConfig implements TuningConfig, AppenderatorConfig
 {
   private static final int defaultMaxRowsInMemory = 75000;
+  private static final int defaultMaxOccupationInMemory = 64 << 20;
```

Comment:
Any reason for this default? It seems small, perhaps default it to 1/4 of the heap size instead.